data/json/recipes/recipe_food.json

```diff
@@ -7693,6 +7693,18 @@
   ]
 },{
   "type" : "recipe",
+  "result": "vodka",
```

Comment:
Should be moonshine, not vodka. Vodka is a consumption-quality alcohol with "fun" stat of one, not ethanol+water.